openmp/libomptarget/unittests/CMakeLists.txt

```
@@ -0,0 +1,8 @@
+add_custom_target(LibomptUnitTests)
+set_target_properties(LibomptUnitTests PROPERTIES FOLDER "Libomptarget tests")
```

```suggestion
set_target_properties(LibomptUnitTests PROPERTIES FOLDER "Tests/UnitTests")
```
This is a weird folder name.